.github/workflows/osrm-backend.yml

```diff
@@ -264,83 +264,7 @@ jobs:
             CXXCOMPILER: g++-8
             CXXFLAGS: -Wno-cast-function-type
 
-          - name: conan-macos-x64-release-node-16
```

Comment:
This PR allowed to get rid from this additional jobs, but at the same time we can now be compatible with more Node versions.